Code/Engine/RendererVulkan/Resources/Implementation/BufferVulkan.cpp

```diff
@@ -190,11 +198,11 @@ void ezGALBufferVulkan::SetDebugNamePlatform(const char* szName) const
   m_pDeviceVulkan->SetDebugName(szName, m_currentBuffer.m_buffer, m_currentBuffer.m_alloc);
 }
 
-vk::DeviceSize ezGALBufferVulkan::GetAlignment(const ezGALDeviceVulkan* pDevice, vk::BufferUsageFlags usage) const
+vk::DeviceSize ezGALBufferVulkan::GetAlignment(const ezGALDeviceVulkan* pDevice, vk::BufferUsageFlags usage)
 {
   const vk::PhysicalDeviceProperties& properties = pDevice->GetPhysicalDeviceProperties();
 
-  vk::DeviceSize alignment = 4;
+  vk::DeviceSize alignment = ezMath::Max(4ull, properties.limits.nonCoherentAtomSize);
```

Comment:
`ezMath::Max<vk::DeviceSize>(4, properties.limits.nonCoherentAtomSize);` to make it cross-platform vk::DeviceSize is `unsigned long int` on linux but `unsigned long long int` on windows. So so `ull` only works on windows.

Reply:
Oh didn't know that. Fixed.